frontend/src/app/components/FeaturedExperiment/FeaturedButton.js

```diff
@@ -42,18 +42,56 @@ export default class FeaturedButton extends React.Component {
   };
 
   renderLegalLink() {
+    const { sendToGA, eventCategory, hasAddon, installed,
+      experiment } = this.props;
     const launchLegalModal = (ev) => {
       ev.preventDefault();
       this.setState({ showLegalDialog: true });
+      sendToGA("event", {
+        eventCategory,
+        eventAction: "link click",
+        eventLabel: "Popup Featured privacy",
+        dimension1: hasAddon,
+        dimension2: Object.keys(installed).length > 0,
+        dimension3: Object.keys(installed).length,
+        dimension11: experiment.slug,
+        dimension13: "Featured Experiment"
+      }, ev);
+    };
+
+    const sendTermsMetric = (ev) => {
+      sendToGA("event", {
+        eventCategory,
+        eventAction: "link click",
+        eventLabel: "Open general terms",
+        dimension1: hasAddon,
+        dimension2: Object.keys(installed).length > 0,
+        dimension3: Object.keys(installed).length,
+        dimension11: experiment.slug,
+        dimension13: "Featured Experiment"
+      }, ev);
+    };
+
+    const sendPrivacyMetric = (ev) => {
+      sendToGA("event", {
+        eventCategory,
+        eventAction: "link click",
+        eventLabel: "Open general privacy",
+        dimension1: hasAddon,
+        dimension2: Object.keys(installed).length > 0,
+        dimension3: Object.keys(installed).length,
+        dimension11: experiment.slug,
+        dimension13: "Featured Experiment"
+      }, ev);
```

Comment:
There's a lot of duplication in the above three methods. It might make sense to have a single currying function that does the majority of the work.